src/XrdHttp/XrdHttpReq.hh

```diff
@@ -116,6 +116,14 @@ private:
   
   // Sanitize the resource from http[s]://[host]/ questionable prefix
   void sanitizeResourcePfx();
+
+   /**
+   * Select the checksum to be computed depending on the userDigest passed in parameter
+   * @param userDigest the digest request from the user (extracted from the Want-Digest header)
+   * @param selectedChecksum the checksum that will be performed
+   */
+  void selectChecksum(const std::string & userDigest, std::string & selectedChecksum);
```

Comment:
Is this a private class in XRootD? I wonder if adding this won't require users to recompile (meaning that it should go into 5.6).

Reply:
I think this is OK as the method aded is non-virtual and the whole set of classes is in one library which gets replaced in any release update. So, it should not require recompilation of anything that was created by a user.